es-ds-styles/scss/utilities/_position.scss

```diff
@@ -32,3 +32,11 @@
     z-index: variables.$zindex-sticky;
   }
 }
+
+// Used with position-absolute to center an element within it's container
+.abs-center {
+    left: 50%;
+    top: 50%;
+    transform: translateX(-50%) translateY(-50%);
+    z-index: variables.$zindex-overlay;
```

Comment:
Hmm 1035 is pretty high - I _think_ that because this is within the position relative context of the container, that forms a separate stacking context from all other z-indices on the page, meaning the play icon might not randomly appear on top of a modal, for example, but I'm not 100% sure.

Reply:
I decoupled the z-index from `abs-center` and added a modal prop to `es-video`. I think this addresses the issue if I understand correctly.

Reply:
A couple quick tests, and it looks like things behave as expected without using `variables.$z-index-modal-overlay`, going to revert that bit.

<img width="676" alt="Screenshot 2024-10-03 at 12 40 39 PM" src="https://github.com/user-attachments/assets/f983fc0a-ea51-4ba1-bccc-65a313d18dfb">

<img width="687" alt="Screenshot 2024-10-03 at 12 40 55 PM" src="https://github.com/user-attachments/assets/dfef7ddd-9aa0-4157-8213-9d2b321bff20">

Reply:
Yea that's why I ended up using https://loremflickr.com/200/200?random=3 🙁